### ISSUE TITLE
[gui error report] Error: Initializing daemon...

### ISSUE THREAD

**Issue by timebankspace** [ipfs/ipfs-desktop#2179]
👉️ Please describe what you were doing when this error happened.

**Specifications**

- **OS**: darwin
- **IPFS Desktop Version**: 0.21.0
- **Electron Version**: 19.0.4
- **Chrome Version**: 102.0.5005.63

**Error**

```
Error: Initializing daemon...
go-ipfs version: 0.13.0
Repo version: 12
System version: amd64/darwin
Golang version: go1.18.3
Found outdated fs-repo, migrations need to be run.
Looking for suitable migration binaries.
Need 1 migrations, downloading.
Downloading migration: fs-repo-11-to-12...
Fetching with HTTP: "https://ipfs.io/ipfs/QmdaCHYBDHEhXCMoynH5UcohEay6m1XayZCcxWZzKAHNVN/fs-repo-11-to-12/versions"
Fetching with HTTP: "https://ipfs.io/ipfs/QmdaCHYBDHEhXCMoynH5UcohEay6m1XayZCcxWZzKAHNVN/fs-repo-11-to-12/versions"
Fetching with HTTP: "https://ipfs.io/ipfs/QmdaCHYBDHEhXCMoynH5UcohEay6m1XayZCcxWZzKAHNVN/fs-repo-11-to-12/versions"
Error fetching: exceeded number of retries. last error was http.DefaultClient.Do error: Get "https://ipfs.io/ipfs/QmdaCHYBDHEhXCMoynH5UcohEay6m1XayZCcxWZzKAHNVN/fs-repo-11-to-12/versions": dial tcp 199.96.58.15:443: i/o timeout
Fetching with IPFS: "fs-repo-11-to-12/versions"
Fetching with HTTP: "https://ipfs.io/ipfs/QmdaCHYBDHEhXCMoynH5UcohEay6m1XayZCcxWZzKAHNVN/fs-repo-11-to-12/v1.0.2/fs-repo-11-to-12_v1.0.2_darwin-amd64.tar.gz"
Fetching with HTTP: "https://ipfs.io/ipfs/QmdaCHYBDHEhXCMoynH5UcohEay6m1XayZCcxWZzKAHNVN/fs-repo-11-to-12/v1.0.2/fs-repo-11-to-12_v1.0.2_darwin-amd64.tar.gz"
Fetching with HTTP: "https://ipfs.io/ipfs/QmdaCHYBDHEhXCMoynH5UcohEay6m1Xa

**Comment by hacdias** [comment]
Closing in favor of #2177.

> When upgrading, IPFS may need to perform migrations and for that we need a stable connection to download the required information for the migrations. Sometimes, the Internet connection may fail or be blocked by firewalls or antivureses and then you will run into a network error. Before submitting an issue, please try the following:
> 
> 1. Check if you are connected to the Internet;
> 2. Make sure your firewall or antivirus is not blocking requests, such as P2P traffic;
> 3. Try again, by restarting IPFS Desktop.
